planner/core/logical_plan_builder.go

```diff
@@ -4651,9 +4651,7 @@ func (b *PlanBuilder) buildMemTable(_ context.Context, dbName model.CIStr, table
 	}.Init(b.ctx, b.getSelectOffset())
 	p.SetSchema(schema)
 	p.names = names
-	for i, col := range tableInfo.Columns {
-		p.Columns[i] = col
-	}
+	copy(p.Columns, tableInfo.Columns)
```

Comment:
Unrelated change?

Reply:
linter upgrading for Golang 1.18 generics.